code/game/gamemodes/handofgod/god.dm

```diff
@@ -8,7 +8,8 @@
 	see_in_dark = 0
 	see_invisible = 55
 	sight = SEE_TURFS | SEE_MOBS | SEE_OBJS | SEE_SELF
-	languages = HUMAN | MONKEY | ALIEN | ROBOT | SLIME | DRONE | SWARMER
+	languages_spoken = HUMAN | MONKEY | ALIEN | ROBOT | SLIME | DRONE | SWARMER
```

Comment:
well ```ALL``` sounds like it's gonna get used for ANYTHING, which worries me people will assume it's unused, and thus define their own ```ALL```'s, thus breaking all the ... ```ALL```s.